src/epub/text/chapter-22.xhtml

```diff
@@ -28,7 +28,7 @@
 			<p>“What answer will there be?”</p>
 			<p>“The answer will be as God pleases.⁠ ⁠… Go!”</p>
 			<p>The messengers rose and went away, and Hadji Murád continued to sit on the carpet, leaning his elbows on his knees. He sat thus a long time, and pondered.</p>
-			<p>“What am I to do? To take Shamil at his word and return to him?” he thought. “He is a fox and will deceive me. Even if he did not deceive me, it would still be impossible to submit to that red liar. It is impossible⁠ ⁠… because now that I have been with the Russians he will not trust me,” thought Hadji Murád; and he remembered a Tavlinian fable about a falcon who had been caught and lived among men and afterwards returned to his own kind in the hills. He returned, wearing jesses with bells; and the other falcons would not receive him. “Fly back to where they hung those silver bells on thee!” said they. “We have no bells and no jesses.” The falcon did not want to leave his home, and remainedl but the other falcons did not wish to let him stay there, and pecked him to death.</p>
+			<p>“What am I to do? To take Shamil at his word and return to him?” he thought. “He is a fox and will deceive me. Even if he did not deceive me, it would still be impossible to submit to that red liar. It is impossible⁠ ⁠… because now that I have been with the Russians he will not trust me,” thought Hadji Murád; and he remembered a Tavlinian fable about a falcon who had been caught and lived among men and afterwards returned to his own kind in the hills. He returned, wearing jesses with bells; and the other falcons would not receive him. “Fly back to where they hung those silver bells on thee!” said they. “We have no bells and no jesses.” The falcon did not want to leave his home, and remained but the other falcons did not wish to let him stay there, and pecked him to death.</p>
```

Comment:
This should actually be a semicolon, not an `l`

Reply:
can confirm l should be a semicolon.
Sorry, for not properly checking in scans.

Reply:
Can you correct the pull request?